subprojects/tooling-api/src/main/java/org/gradle/tooling/events/test/TestOutputEvent.java

```diff
@@ -14,12 +14,17 @@
  * limitations under the License.
  */
 
-package org.gradle.tooling.internal.protocol.events;
+package org.gradle.tooling.events.test;
+
+import org.gradle.tooling.events.ProgressEvent;
 
 /**
- * DO NOT CHANGE THIS INTERFACE. It is part of the cross-version protocol.
+ * An event that informs about a test having started its execution.
  *
  * @since 6.0
  */
-public interface InternalTestOutputStartedProgressEvent extends InternalOperationStartedProgressEvent {
+public interface TestOutputEvent extends ProgressEvent {
```

Comment:
I think this should be a `TestProgressEvent` so clients now which test it belongs to.

Reply:
Make sense. But the clients can already access the test details by looking up the parent operation via `ProgressEvent.getParent()`.

Reply:
I've added a test to demonstrate how to connect the test output with the corresponding test event.